packages/fxa-graphql-api/src/decorators.ts

```diff
@@ -45,6 +45,13 @@ export const GqlXHeaders = createParamDecorator(
     // We don't need to propagate the authorization token because
     // it needs to be updated to originate from gql server.
     delete headers['authorization'];
+    
+    // Set the x-forwarded-for header since the auth-server will use this
+    // to determine client geolocation
+    if (ctx.req?.ip) {
+     headers['x-forwarded-for'] = ctx.req?.ip;
```

Comment:
Just to be explicit here, `ctx.req.ip` does not come from a user-agent, right?

Reply:
Good call out, this value is populated from Graphql request and not injected.